workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php

```diff
@@ -6,10 +6,13 @@
  *
  * Show Usages (uses popup)
  *
- * Ctrl+Alt+F7 (Windows/Linux)
+ * Ctrl+Alt+7 (Windows/Linux)
  * Alt+Command+F7 (macOS)
  *
  * Highlight usages of a given symbol in the current project.
+ *
+ * NOTE: On Ubuntu 18.04 and newer, Alt+F7 is overwritten
```

Comment:
Thank you! This information is indeed useful!

Reply:
Thanks for pointing that out :+1: